airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/discover/DiscoverOperation.kt

```diff
@@ -66,9 +48,7 @@ class DiscoverOperation<T : ConfigurationSpecification, C : DestinationConfigura
     }
 
     private fun handleException(t: Throwable) {
-        val (traceMessage, statusMessage) = exceptionHandler.handleCheckFailure(t)
-        outputConsumer.accept(traceMessage)
-        outputConsumer.accept(statusMessage)
+        outputConsumer.accept(exceptionHandler.handle(t))
```

Comment:
Not really related to what I was trying to achieve but this seems like the error handling is a copy/paste of check but it does not make sense to emit a connection status message during discover